### PR TITLE
updated PPP

### PR DESCRIPTION
**Pull request by yongmingyang**

### CODE REVIEW

File: docs/team/yongmingyang.md [hunk 1]
```diff
@@ -9,45 +9,40 @@ Medmoriser is a desktop app to help medical students organise, memorise and revi
...
+        3. [#68](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/68)
+        4. [#75](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/75)
+        5. [#81](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/81)
+        6. [#138](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/138)
+        7. [#139](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/139)
     * Credits: The find functionality makes use of code from the following links:
         1. https://stackoverflow.com/questions/36793397/check-if-string-contains-word-not-substring.
-
+  * Refactoring instances of `Address` to `Answer`, `UniquePersonList` to `UniqueQuestionSetList`, `AddressBook` to `Medmoriser` and `Person` to `QuestionSet` (later changed to QAndA by Jian Ling): [#36](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/36)
+  * Updated GUI, for quizing part in particular: [#165](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/165), [#172](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/172), [#46](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/46)
```
Comment (jonfoocy) [inline docs/team/yongmingyang.md]: Might want to list the PRs in ascending order

File: docs/team/yongmingyang.md [hunk 2]
```diff
@@ -9,45 +9,40 @@ Medmoriser is a desktop app to help medical students organise, memorise and revi
...
-  * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
-  * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
-
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
-
-* _{you can add/remove categories in the list above}_
+  * PRs reviewed (with non-trivial review comments): [#38](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/38), [#62](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/62)
+  * Contributed to forum discussions (examples: [#127](https://github.com/nus-cs2103-AY2021S1/forum/issues/127), [#349](https://github.com/nus-cs2103-AY2021S1/forum/issues/349), [#128](https://github.com/nus-cs2103-AY2021S1/forum/issues/128)
```
Comment (jonfoocy) [inline docs/team/yongmingyang.md]: Might want to list the issues in ascending order

**Review by jonfoocy — APPROVED** [review]
Just some suggestions to consider

File: docs/team/yongmingyang.md (hunk 1 above)
Comment (yongmingyang) [reply docs/team/yongmingyang.md]: aight

**Review by tengjianling — APPROVED** [review]

**Pull request merged**